compose_generator.py

```diff
@@ -216,6 +226,15 @@ def monitoring_system_handler(config_dict):
                     }
                 }
             }
+        },
+        "networks": {
+            "static-network": {
+                "ipam": {
+                    "config": [
+                        {"subnet": "192.169.30.0/16"}
```

Comment:
take the subnet from the requested url in the conf file

Reply:
I changed it so the subnet will be generated from the requested ip in the config file